src/Employer/Employer.Web/Configuration/CookieNames.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Esfa.Recruit.Employer.Web.Configuration
+{
+    public static class CookieNames
+    {
+        public const string RecruitData = "recruit-data";
+        public const string RecruitTempData = "recruit-temp-data";
+        public const string AntiForgeryCookie = "recruit-x-csrf";
+        public const string SetupEmployer = "setup_employer_{0}";
```

Comment:
Change the underscores to hyphens if you want. I only chose underscores as that that's what the "seen_cookie_message" is using.

Reply:
👍